lib/rom/sql/gateway.rb

```diff
@@ -247,4 +251,6 @@ def transaction_runner(_)
       end
     end
   end
+
+  Configuration.register_event("configuration.gateway.db.connected")
```

Comment:
@ianks could we call this `"configuration.gateway.connected"` instead?

Reply:
yes